k8s/namespaces/ia/ia-aip-frontend/demo.yaml

```diff
@@ -6,7 +6,7 @@ spec:
   values:
     nodejs:
       ingressHost: immigration-appeal.demo.platform.hmcts.net
-      image: hmctspublic.azurecr.io/ia/aip-frontend:pr-747-afc28d8-20220412140117 #{"$imagepolicy": "flux-system:demo-ia-aip-frontend"}
+      image: hmctspublic.azurecr.io/ia/aip-frontend:pr-747-5558459-1b5c47ff780d39
```

Comment:
@rajeshthuraiyur has this been attempted to resolve using the Jenkins build with the correct tagging prior to this PR? (as per our call)

Reply:
@thomast1906 Gregorio is going to work on updating the correct tagging as well. Will revert back this as quickly as we can.